docs/Algorithms.md

```diff
@@ -12,7 +39,7 @@ Each compression method receives its own hyperparameters that are organized as a
   - Automatic bitwidth assignment based on HAWQ
   - Automatic quantization parameter selection and activation quantizer setup based on HW config preset
   - Automatic bitwidth assignment mode AutoQ, based on HAQ, a Deep Reinforcement Learning algorithm to select best mixed precision given quality metric and HW type.
-- [Sparsity](./compression_algorithms/Sparsity.md)
+- [Sparsity](./usage/training_time_compression/other_algorithms/Sparsity.md) (PyTorch,TensorFlow)
```

Comment:
maybe better to call it Unstructured Sparsity

Reply:
Done